src/sandbox-client.ts

```diff
@@ -294,21 +310,21 @@ export class SandboxClient {
    * any sandbox/template created on codesandbox.io, even your own templates) or don't pass
    * in anything and we'll use the default universal template.
    *
-   * This function will also start & connect to the VM of the created sandbox, and return a {@link Sandbox}
-   * that allows you to control the VM.
+   * This function will also start & connect to the VM of the created sandbox as a ROOT session, and return a {@link Sandbox}
```

Comment:
I don't think we should call it `ROOT` because that implies that the user will have `root` access, which is not the case. I would call it a global session.
```suggestion
   * This function will also start & connect to the VM of the created sandbox as the global session, and return a {@link Sandbox}
```